issuance/issuance_test.go

```diff
@@ -701,6 +701,14 @@ func TestIssueCTPoison(t *testing.T) {
 	test.AssertDeepEquals(t, cert.Extensions[8], ctPoisonExt)
 }
 
+func mustDecodeB64(b string) []byte {
```

Comment:
If you add this generic function as a utility somewhere:

```
func Must[T any](t T, err error) T {
	if err != nil {
		panic(err)
	}
	return t
}
```

Then you can just `util.Must(base64.StdEncoding.DecodeString(b))`, and reuse Must elsewhere -- there's a handful of `mustFoo` helpers in boulder already

Reply:
Same! Made a separate PR: https://github.com/letsencrypt/boulder/pull/6955